assets/scripts/book.js

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable func-names */
+/* eslint-disable camelcase */
+/* eslint-disable no-alert */
+/* eslint-disable array-callback-return */
+/* eslint-disable  consistent-return */
+
+
+const addnote = document.querySelector('.add_note');
+const formcontainer = document.querySelector('.form-container');
+
+const form = document.querySelector('form');
+const button = document.querySelector('button');
+let book_list = [];
+function Book(title, name, pages, readstatus = false) {
+  this.title = title;
+  this.name = name;
+  this.pages = pages;
+  this.readstatus = readstatus;
+}
+
+let read = '';
+
+Book.prototype.info = function () {
+  if (this.readstatus) {
+    read = 'Read!';
+    return read;
+  }
+  read = 'Not Read!';
+  return read;
+};
+Book.prototype.toggleStatus = function () {
+  this.readstatus = !this.readstatus;
+};
+
+
+if (localStorage.length < 1) {
+  book_list = [
+    {
+      title: 'Lord of the Rings', name: 'JRR Tolkien', pages: 300, readstatus: true,
+    },
+    {
+      title: 'Neuromante', name: 'William Gibson', pages: 200, readstatus: true,
+    },
+    { title: 'The Call of The Wild', name: 'Jack London', pages: '80' },
+    {
+      title: "Hitchiker's Guide to The Galaxy", name: 'Douglas Adams', pages: '250', readstatus: true,
+    },
+  ];
+} else {
+  const stored_list = localStorage.getItem('book_list');
+  book_list = JSON.parse(stored_list);
```

Comment:
Here too. Kindly check all others as well. Ruby uses snake_case, JavaScript uses camelCase.
```suggestion
  bookList = JSON.parse(stored_list);
```